### PR TITLE
install fish

### PR DESCRIPTION
**Pull request by runamoon**
### TODO
change to fish shell

### CODE REVIEW

File: roles/fish/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  become: yes
+  shell: chsh -s /usr/bin/fish
+
+- name: register config.fish
+  stat: path=~/.config/fish/config.fish
+  register: config_fish
+
+- name: touch config.fish
+  when: config_fish.stat.md5 is not defined
+  file: path=~/.config/fish/config.fish state=touch
```
Comment (ojiry) [inline roles/fish/tasks/main.yml]: Would you like arrange config.fish in templates?
config.fish is automatically loading local_config.fish.

File: site.yml [hunk 2]
```diff
@@ -8,3 +8,4 @@
 - hosts: all
   roles:
     - golang
+    - fish
```
Comment (ojiry) [inline site.yml]: fish is above golang because the same layer.

File: roles/user/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+
+- name: update user and set password
+  action: user name=ubuntu password=ubuntu shell=/usr/bin/fish
```
Comment (ojiry) [inline roles/user/tasks/main.yml]: Use action shorthand.
http://docs.ansible.com/ansible/playbooks_intro.html#action-shorthand

File: roles/fish/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+- name: install fish
+  apt:
+    pkg: fish
+    update_cache: yes
+    force: yes
+    state: present
+  become: yes
+
+- name: change to fish
```
Comment (ojiry) [inline roles/fish/tasks/main.yml]: Is this unnecessary code?

File: roles/user/tasks/main.yml (hunk 3 above)
Comment (runamoon) [inline roles/user/tasks/main.yml]: Thank you so much!
Nice documentation.

**Comment by runamoon** [comment]
```
> vagrant ssh
Welcome to Ubuntu 16.04 LTS (GNU/Linux 4.4.0-22-generic x86_64)

 * Documentation:  https://help.ubuntu.com/

  Get cloud support with Ubuntu Advantage Cloud Guest:
    http://www.ubuntu.com/business/services/cloud

56 packages can be updated.
29 updates are security updates.


Last login: Sun Jun 12 12:32:21 2016 from 10.0.2.2
Welcome to fish, the friendly interactive shell
Type help for instructions on how to use fish
ubuntu@ubuntu-xenial ~> 
```

**Comment by ojiry** [comment]
Please rebase and compressed commit.

**Comment by ojiry** [comment]
Nice rebase 🙆

**Pull request merged**